website/config.toml

```diff
@@ -113,10 +113,8 @@ algolia_docsearch = false
 
 # User interface configuration
 [params.ui]
-# Enable to show the side bar menu in its compact state.
-sidebar_menu_compact = true
 #  Set to true to disable breadcrumb navigation.
-breadcrumb_disable = true
+breadcrumb_disable = false
```

Comment:
I like this, but for some reason it starts with "1. "

![Screenshot from 2020-10-19 09-29-15](https://user-images.githubusercontent.com/1028657/96457596-fa592680-11ed-11eb-96a5-fadfa18ed8e7.png)

Reply:
@asmacdo agreed. Fyi we have a follow up PR https://github.com/operator-framework/operator-sdk/pull/4004 that takes care of this along with a few other nits. 

Eg. in https://olm.operatorframework.io/docs/tasks/list-operators-available-to-install/ you won't see the "1" which is what that ^ PR does.